docs/my-classic-app-with-lazy-loaded-translations/app/routes/application.ts

```diff
@@ -15,7 +16,19 @@ export default class ApplicationRoute extends Route {
   }
 
   private async loadTranslations(locale: 'de-de' | 'en-us') {
-    const response = await fetch(`/translations/${locale}.json`);
+    const filePath = `translations/${locale}.json`;
+    let resource: string;
+
+    if (ENV.environment === 'production') {
+      const response = await fetch('/assets/assetMap.json');
+      const assetMap = await response.json();
+
+      resource = `/${assetMap.assets[filePath]}`;
```

Comment:
We can locally test this conditional block by passing `enabled: true` to the `fingerprint` option, then changing the condition to `ENV.environment === 'development'`.